distro/otel.go

```diff
@@ -136,8 +142,20 @@ func newResource(ctx context.Context) (*resource.Resource, error) {
 	}
 
 	// Add Splunk-specific attributes.
-	splunkRes := resource.NewSchemaless(attribute.String(distroVerAttr, Version()))
-	res, err = resource.Merge(res, splunkRes)
+	splunkVerRes := resource.NewSchemaless(attribute.String(splunkVerAttr, Version()))
+	res, err = resource.Merge(res, splunkVerRes)
+	if err != nil {
+		return nil, err
+	}
+
+	distroVerRes := resource.NewSchemaless(attribute.String(distroVerAttr, Version()))
+	res, err = resource.Merge(res, distroVerRes)
+	if err != nil {
+		return nil, err
+	}
+
+	distroNameRes := resource.NewSchemaless(attribute.String(distroNameAttr, Name()))
+	res, err = resource.Merge(res, distroNameRes)
```

Comment:
There is no need to merge it 3 times. https://pkg.go.dev/go.opentelemetry.io/otel/sdk/resource#NewSchemaless is a [variadic function](https://gobyexample.com/variadic-functions)